website/src/docs/uppy.md

```diff
@@ -711,6 +711,10 @@ this.info({
 
 Subscribe to an uppy-event. See below for the full list of events.
 
+### `uppy.once('event', action)`
+
+Subscribe once to an uppy-event. See below for the full list of events.
```

Comment:
```suggestion
Create en event listener that fires once. See below for the full list of events.
```

Reply:
Ah, I was thinking of using the description from `namespace-emitter` docs but felt it was inconsistent with the existing stuff. This is definitely clearer.